spec/tests/ladybug_energy_model_model_spec.rb

```diff
@@ -53,12 +53,14 @@
     expect(extension.schema_validation_errors.empty?).to be true
   end
  
+  #add assertions
   it 'can load and validate example face by face model' do
     file = File.join(File.dirname(__FILE__), '../files/example_face_by_face_model.json')
     model = Ladybug::EnergyModel::Model.read_from_disk(file) 
     expect(model.valid?).to be true
     expect(model.validation_errors.empty?).to be true 
-    model.to_openstudio_model
+    face_model = model.to_openstudio_model
+    expect(face_model.getFaceByName).not_to be nil
```

Comment:
OpenStudio Model's don't have a Face class, the class name is Surface.  You could call `openstudio_model.getSurfaceByName('name')` but you have to pass in the name of the object as input

Reply:
Also, `getSurfaceByName` will return an empty optional if the object does not exist in the model.  Due to Ruby quirks, this is not the same as nil.  You would have to test:

`expect(face_model.getSurfaceByName('name').empty?).not_to be true`